resources/js/features/game-list/components/WantToPlayGamesDataTable/DataTableSearchInput/DataTableSearchInput.tsx

```diff
@@ -0,0 +1,79 @@
+import type { Table } from '@tanstack/react-table';
+import { useEffect, useState } from 'react';
+import { useDebounce } from 'react-use';
+
+import { BaseInput } from '@/common/components/+vendor/BaseInput';
+import { cn } from '@/utils/cn';
+
+import { useSearchInputHotkey } from './useSearchInputHotkey';
+
+interface DataTableSearchInputProps<TData> {
+  table: Table<TData>;
+}
+
+export function DataTableSearchInput<TData>({ table }: DataTableSearchInputProps<TData>) {
+  const initialValue = (table.getColumn('title')?.getFilterValue() as string) ?? '';
+
+  const [rawInputValue, setRawInputValue] = useState(initialValue);
+
+  const { hotkeyInputRef } = useSearchInputHotkey({ key: '/' });
+
+  /**
+   * Listen for changes with column filter state and stay in sync. Otherwise,
+   * when the user presses the "Reset" button to reset all filters, our search
+   * value will remain. It needs to be reset too.
+   */
+  useEffect(() => {
+    const filterValue = (table.getColumn('title')?.getFilterValue() as string) ?? '';
+    setRawInputValue(filterValue);
+    // eslint-disable-next-line react-hooks/exhaustive-deps -- this is a valid dependency array
+  }, [table.getState().columnFilters]);
+
+  /**
+   * Wait until the user is done typing before we fetch from the back-end.
+   */
+  useDebounce(
+    () => {
+      if (rawInputValue.length >= 3 || rawInputValue.length === 0) {
+        table.getColumn('title')?.setFilterValue(rawInputValue);
+      }
+    },
+    200,
+    [rawInputValue],
```

Comment:
Should explicit wildcards be allowed?
![image](https://github.com/user-attachments/assets/6a6f47b5-523c-4813-af79-c1dee4c8cfb6)

I guess the big search bar does it too.

Reply:
No strong feeling from me either way on this. I haven't changed anything in latest, but happy to proceed with attacking this if you'd like to see it changed.

Reply:
It mimics the main search bar. As long as we can't do SQL injection, I'm fine with it

Reply:
Should be safe from SQL prevention, as everything is passing through Eloquent ORM and `Request`.